content/actions/creating-actions/creating-a-javascript-action.md

```diff
@@ -176,6 +176,14 @@ git tag -a -m "My first action release" v1
 git push --follow-tags
 ```
 
+---
+
+**NOTE**
+
+Checking in `node_modules` into a git repository is known to cause a [few problems](https://byjoeybaker.com/why-you-should-never-commit-node-modules-in-nodejs). Hence the following alternative approach exists
+
+---
+
 As an alternative to checking in your `node_modules` directory you can use a tool called [`@vercel/ncc`](https://github.com/vercel/ncc) to compile your code and modules into one file used for distribution.
```

Comment:
Let's simplify this to reduce cognitive load on the reader. I'd be happy to include a link, following the [style guidelines](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#links), if you can provide a more authoritative source. 

```suggestion
Checking in your `node_modules` directory can cause problems. As an alternative, you can use a tool called [`@vercel/ncc`](https://github.com/vercel/ncc) to compile your code and modules into one file used for distribution.
```